website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -27,6 +27,13 @@ This topic describes the commands, subcommands, and available options for using
    mv bin/consul-k8s /usr/local/bin/consul-k8s
    ```
 
+1. (Optional) Issue the `consul-k8s version` command to verify the installation:
+
+   ```shell-session
+   bin/consul-k8s version
+   consul-k8s 0.34.1-dev
+   ```
```

Comment:
If you've already `mv`'d it then `bin/consul-k8s` will fail so I think we just need the second line?

Reply:
ah, thanks, good catch